devicon.json

```diff
@@ -3455,6 +3455,26 @@
         "color": "#E24329",
         "aliases": []
     },
+    {
+        "name": "gitlabci",
+        "altnames": [],
+        "tags": [
+            "continuous-delivery",
+            "devops",
+            "integration"
+        ],
+        "versions": {
+            "svg": [
+                "original-wordmark",
+                "plain-wordmark"
```

Comment:
```suggestion
                "original-wordmark"
```

Reply:
Why remove it? I believe `plain-wordmark` counts as an SVG as well. 🤔

Reply:
I believe that `original`, `plain` and `line` versions have priority over their `wordmark` versions. Therefore, I think we should rename the files and add aliases to their `wordmark` versions.
|Current file name|New file name|
|:-:|:-:|
|gitlabci-original-wordmark.svg|gitlabci-original.svg|
|gitlabci-plain-wordmark.svg|gitlabci-plain.svg|

Then:
```json
"versions": {
    "svg": [
        "original",
        "plain"
    ],
    "font": [
        "plain"
    ]
}
...
"aliases": [
    {
        "base": "original",
        "alias": "original-wordmark"
    },
    {
        "base": "plain",
        "alias": "plain-wordmark"
    }
]
```
> *Obs: `...` should not be included in `devicon.json` file.*

I also think this will not happen only in a scenario that we have both `original`, `plain` and their `wordmark` versions. In other words, each one goes to `versions.svg`, `plain` and `plain-wordmark` to `versions.font` and no `aliases`.